packages/postcss/package.json

```diff
@@ -18,20 +18,17 @@
     "url": "https://github.com/xing/hops.git"
   },
   "dependencies": {
-    "core-js": "^3.2.1",
-    "css-loader": "^3.2.0",
-    "enhanced-resolve": "^5.0.0",
+    "css-loader": "^5.0.1",
     "hops-mixin": "^14.0.0-nightly.4",
     "mini-css-extract-plugin": "^1.0.0",
-    "optimize-css-assets-webpack-plugin": "^5.0.3",
+    "optimize-css-assets-webpack-plugin": "^5.0.4",
     "pathifist": "^1.0.0",
-    "postcss-import": "^12.0.1",
-    "postcss-loader": "^3.0.0",
-    "postcss-modules-extract-imports": "^2.0.0",
+    "postcss": "^8.2.1",
```

Comment:
Added `postcss` here explicitly. This shouldn't be necessary since the `css-loader` brings it, too. But `postcss-preset-env` isn't `postcss@8`-ready, yet, and still comes along with `postcss@7`. So this dependency is in here to force the correct version. :feelsgood: